packages/kit/src/packaging/index.js

```diff
@@ -104,6 +106,29 @@ export async function make_package(config, cwd = process.cwd()) {
 	}
 
 	pkg.exports = { ...generated, ...pkg.exports };
+
+	if (contains_svelte_files) {
```

Comment:
Should we ignore if the user already set the `pkg.svelte` field?
```suggestion
	if (!pkg.svelte && contains_svelte_files) {
```